### PR TITLE
[Candidacy] fix update photo

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Entity/ImageTrait.php [hunk 1]
```diff
@@ -23,9 +24,7 @@ public function setImageName(?UploadedFile $image): void
                 md5(
                     sprintf(
                         '%s@%s',
-                        method_exists($this, 'getUuid')
-                            ? $this->getUuid()->toString()
-                            : \strval($this->getId()),
+                        method_exists($this, 'getUuid') ? $this->getUuid()->toString() : ($this->getId() ?? Uuid::uuid4()),
```
Comment (Remg) [inline src/Entity/ImageTrait.php:27]: :trophy:

File: src/VotingPlatform/Events.php [hunk 2]
```diff
@@ -5,5 +5,6 @@
 final class Events
 {
     public const CANDIDACY_CREATED = 'candidacy.created';
+    public const CANDIDACY_UPDATE = 'candidacy.update';
```
Comment (Remg) [inline src/VotingPlatform/Events.php:8]: candidacy.updated ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -138,7 +138,7 @@ jobs:
                 set -x
 
                 mkdir -p ./behat
-                make -j2 tfp-rabbitmq tfp-db
+                make -j2 tfp-rabbitmq tfp-db perm
```
Comment (kevin-verschaeve) [inline .circleci/config.yml:141]:
```suggestion
                make -j3 tfp-rabbitmq tfp-db perm
```
?

Comment (ottaviano) [reply .circleci/config.yml]: I've changed this by doing `make perm` after `make -j2 tfp-rabbitmq tfp-db`

**Comment by ottaviano** [comment]
[ch11510]

**Pull request merged**